Apps/APN-UIKit/APN UIKit/APN UIKit/View/Extension/TextFieldExtension.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+import UIKit
+
+extension UITextField {
+        
+    var isTextTrimEmpty: Bool {
```

Comment:
I think that using `String.trim` is really important for getting the String value of a `UITextField`. 

Because of that, I think that instead of this function, we should instead:

1. Replace this extension function with: 
```suggestion
    var trim: String? {
        text?.trimmingCharacters(in: CharacterSet.whitespacesAndNewlines)
    }
```
2. Use this code to do the same thing as this extension function:
```swift
textFieldInstance.trim.isEmpty
```

This is an idea so that we can use `.trim` in the app.